sycl/source/detail/scheduler/commands.hpp

```diff
@@ -38,8 +38,8 @@ enum BlockingT { NON_BLOCKING = 0, BLOCKING };
 
 // The struct represents the result of command enqueueing
 struct EnqueueResultT {
-  enum ResultT { SUCCESS, BLOCKED, FAILED };
-  EnqueueResultT(ResultT Result = SUCCESS, Command *Cmd = nullptr,
+  enum ResultT { SYCL_SUCCESS, SYCL_BLOCKED, SYCL_FAILED };
```

Comment:
I think this enum represents the result of enqueue operation, so using `ENQUEUE_` or `SYCL_ENQUEUE_` prefix instead of generic `SYCL_` seems more suitable.
```suggestion
  enum ResultT { SYCL_ENQUEUE_SUCCESS, SYCL_ENQUEUE_BLOCKED, SYCL_ENQUEUE_FAILED };
```

Reply:
That will work, too.